src/main/java/picard/util/DbSnpBitSetUtil.java

```diff
@@ -213,6 +214,14 @@ private static void loadVcf(final PicardHtsPath dbSnpFile,
 
         final Optional<ProgressLogger> progress = log.map(l -> new ProgressLogger(l, (int) 1e5, "Read", "variants"));
         final VCFFileReader variantReader = new VCFFileReader(dbSnpFile.toPath(), intervals != null);
+        
+        SequenceDictionaryUtils.assertSequenceDictionariesEqual(
+            variantReader.getFileHeader().getSequenceDictionary(),
+            "DBSNP_REFERENCE_VCF",
+            sequenceDictionary,
+            "INPUT_VCF"
```

Comment:
I think this is fine to list the command line parameter names in an internal function

Reply:
Done.